pysmt/shortcuts.py

```diff
@@ -237,6 +237,19 @@ def is_sat(formula, quantified=False, solver_name=None, logic=None):
                               solver_name=solver_name,
                               logic=logic)
 
+def get_model(formula, quantified=False, solver_name=None, logic=None):
+    """ Similar to :py:func:`is_sat` but returns a model if the formula is
+    satisfiable, otherwise None."""
+    env = get_env()
+    if formula not in env.formula_manager:
+        warnings.warn("Warning: Contextualizing formula during is_sat")
```

Comment:
Unless there are multiple managers (e.g., during multi-processing), we do not need this check. 
Also, we do not do it for the other shortcuts. Finally, it might make more sense to do this within the formula manager, instead than in the shortcut.
(note: typo is_sat/get_model)